proposed/cli-acquisition-tool.md

```diff
@@ -0,0 +1,149 @@
+
+### Initial setup
+
+On first-run, `dnup-init` will configure the users environment to support user-local hives. DOTNET_ROOT will be set to point to the `dnup` user-local root, the user's PATH will be appended to, and on Windows `dnup-init` will request to elevate to set a Windows Registry key that will prevent global .NET installers from setting the global .NET Install location on the System PATH (which would overwrite the user PATH always).
+
+### Install locations
+
+These installs will be located at a default _user-local hive_ instead of a globally-reachable system hive. This will allow users to manage new tooling versions without needing to elevate. The tool will support being configured to install to other locations via configuration file, so that simple use of the tool remains consistent.
+
+### Install provenance
+
+The data about installs will be sourced from the release manifests that are published on a regular cadence by the .NET Releases team. These manifests, and the release artifact data and signatures they contain, will enable the tool to confirm the provenance of any toolsets downloaded to the users' computer.
+
+### Update Management
+
+The tool will enable update checks via these same manifests. As hihgly-cacheable static files, they will be easy to check for updates on a regular cadence, and user interactions with the `dnup` tool (directly via a command like `dnup update --check` or similar, or implicitly via IDE or `dotnet` tooling calling such commands) will be managed on a default (but user definable) cadence such as `daily`, `weekly`, etc.
+
+### Self-updates
+
+`dnup` needs to be able to update itself as well, via a command similar to `dnup update --self`. This command will check documented, signed release manifests for `dnup` and download + replace the current `dnup` binary with the new one. Provenance and signature validation of the new binary will also happen to ensure the content is as expected.
+
+## Q & A
+
+### How will I get `dnup`?
+
+We intend to distribute `dnup` via a direct download initially. The dotnet download web site and the Learn documentation for getting started with .NET are the primary candidates. However, it is also common to distribute these kinds of bootstrappers via system package managers, so future efforts could include WinGet packages, Debian and Red Hat packages, a Homebrew cask, etc. The tool itself will be a Native AOT tool that is completely standalone, so acquisition should be relatively simple to set up.
+
+### How will this handle existing global installations?
+
+`dnup` will detect existing global installations but focus on managing user-local installs. Users with existing package manager installations will continue to use those tools. `dnup` will provide guidance when detecting unsupported installation types, and may offer to seed the user-local hive with the same versions so that there are no functionality gaps when starting with `dnup`.
```

Comment:
```suggestion
`dnup` will detect existing global installations but focus on managing user-local installs. Users with existing package manager installations will continue to use those tools. `dnup` will provide guidance when detecting unsupported installation types, and may offer to seed the user-local hive with the same versions so that there are no functionality gaps when starting with `dnup`.
```


I like that you've called this out and thought of a lot of scenarios in the Q&A session. Another important point to me, is how the product clarifies that `dnup uninstall <program files sdk>` would not work, and how we message that - what type of guidance would we provide? A link/doc?